modules/gdscript/gdscript.cpp

```diff
@@ -896,14 +921,15 @@ bool GDScript::_get(const StringName &p_name, Variant &r_ret) const {
 				return true;
 			}
 		}
-
 		{
 			HashMap<StringName, MemberInfo>::ConstIterator E = top->static_variables_indices.find(p_name);
 			if (E) {
 				if (E->value.getter) {
-					Callable::CallError ce;
-					r_ret = const_cast<GDScript *>(this)->callp(E->value.getter, nullptr, 0, ce);
-					return true;
+					const GDScript::MemberInfo *static_member = &E->value;
+					Callable::CallError ce = call_getter(*static_member, p_name, r_ret);
+					if (ce.error == Callable::CallError::CALL_OK) {
+						return true; // TODO: should this just be return ce.error == Callable::CallError::CALL_OK?
+					}
```

Comment:
Makes sense to me. The only problem is that the CI will complain that the call error `ce` is not used. The CI didn't catch this before because `ce` was filled via `call_p`, but now it is returned by my `call_getter` helper. I could just not use my helper function here, but I'm not sure that's the best approach.

Reply:
You could try just calling `call_getter()` without assigning its return value to an unused variable. As far as I remember, the warning about discarding the returned value is not enabled.